cpp/pipelines/remove_dangling_references.cc

```diff
@@ -83,7 +98,7 @@ int Main(int argc, char *argv[]) {
     const auto marc_writer(MARC::Writer::Factory(argv[2]));
     const auto log_file(FileUtil::OpenOutputFileOrDie(argv[3]));
 
-    std::unordered_set<std::string> all_ppns;
+    std::unordered_map<std::string, bool> all_ppns;
```

Comment:
Since it's not only a set or vector of ppns, the name of the variable should also contain something about the meaning of the bool flag.